### DISCUSSION TITLE
VIEW A List or Array inside a key

### DISCUSSION THREAD

**Discussion by giittyy** [mProjectsCode/obsidian-meta-bind-plugin#211] [Q&A] [answered]
In the advanced activity tracker in the example vault it's creating multiple arrays inside the key. But how do I get to those array's in VIEW? Usually I would use a number eg. a.1.b, but that doesn't seem to work. Any suggestions?

**Comment by mProjectsCode** [comment, marked as answer]
try `a[1].b`

**Reply by giittyy** [reply]
That worked thanks
